pkg/terminal/terminal.go

```diff
@@ -243,6 +243,28 @@ func (t *Term) Run() (int, error) {
 		case "nullcmd", "nocmd":
 			commands := cmds.FuzzySearch(strings.ToLower(line))
 			c = append(c, commands...)
+		case "print", "whatis":
+			localVars, err := t.client.ListLocalVariables(
+				api.EvalScope{GoroutineID: -1, Frame: t.cmds.frame, DeferredCall: 0},
+				t.loadConfig(),
```

Comment:
use `api.LoadConfig{}` so we don't load as much, also the trie should be cached at least until a command is executed (i.e. cleared before the promptForInput call).